javascript/jsonwebtoken/security/audit/jwt-exposed-data.yaml

```diff
@@ -11,7 +11,6 @@ rules:
   languages: [javascript]
   patterns:
   - pattern-inside: |
-      ...
```

Comment:
@aryx Could be an interesting thing to detect in your Semgrep rule linter! What code base does that live in and might I take a peek at what it does?

Reply:
Probably worth noting that _sometimes_ this type of pattern can be useful (see jwt-verify, for example)

Reply:
yep totally agree @dlukeomalley . The code of the metachecker is in https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/Metachecking
but it does very little right now.